packages/client/lib/commands/GEOSEARCH.ts

```diff
@@ -22,30 +23,33 @@ export interface GeoSearchByBox {
 
 export type GeoSearchBy = GeoSearchByRadius | GeoSearchByBox;
 
-export function pushGeoSearchArguments(
-  args: CommandArguments,
+export function parseGeoSearchArguments(
+  parser: CommandParser,
   key: RedisArgument,
   from: GeoSearchFrom,
   by: GeoSearchBy,
-  options?: GeoSearchOptions
+  options?: GeoSearchOptions,
+  store?: RedisArgument
```

Comment:
look at this more carefully, as a change in the prototype.

Reply:
removed store from it, as not common, and pushed into its own parseCommand function